appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -423,6 +423,19 @@ public void SelectionIndex(int index) {
     }
   }
 
+  /**
+   */
+   @SimpleFunction(
+      description = "Removes Item from list at a given idnex")
+    public void RemoveItemAtIndex(int index) {
+      if (!dictItems.isEmpty()) {
+        dictItems.remove(index);
+      } else {
+        stringItems.remove(index);
+      }
```

Comment:
Make sure that this logic keeps the two lists in sync. We don't want to remove an element at one index if it has a corresponding representation at the same index in the other list.